packages/xo-web/src/xo-app/backup-ng/new/new-schedule.js

```diff
@@ -9,6 +9,31 @@ import { isEqual } from 'lodash'
 
 import { FormGroup, getRandomId, Input } from './utils'
 
+const Number = [
+  provideState({
+    effects: {
+      onChange: (_, { target: { value } }) => (state, props) => {
+        if (value === '') {
+          return
+        }
+        props.onChange(+value)
+      },
+    },
+    computed: {
+      value: (_, { value }) => value,
```

Comment:
Why use a computed for this?

Reply:
To inject the `value` from the props to the state.

Reply:
And why do you need it to be in the state?